### ISSUE TITLE
Cannot play some transcoded streams

### ISSUE THREAD

**Issue by deece** [MediaBrowser/plugin.video.emby#106]
I have a number of 1080P HEVC .MKV files, with the Kodi Emby plugin configured to transcode. Some refuse to play.

Here is the relevant bit of the log:
```
08:22:41.860 T:18446744072445157928  NOTICE: EMBY.artwork -> DEBUG:: image cache thread count: 25
08:22:41.865 T:18446744072445157928  NOTICE: EMBY.playbackutils -> Play called.
08:22:41.871 T:18446744072445157928  NOTICE: EMBY.playutils -> Can't direct play, play from HTTP enabled.
08:22:41.886 T:18446744072445157928  NOTICE: EMBY.playutils -> Resolution is: 3064P, transcode for resolution: 480P+
08:22:41.886 T:18446744072445157928  NOTICE: EMBY.playutils -> File is transcoding.
08:22:41.910 T:18446744072445157928   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.IndexError'>
                                            Error Contents: list index out of range
                                            Traceback (most recent call last):
                                              File "/mnt/internal_sd/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.emby.tvshows/default.py", line 50, in <module>
                                                entrypoint.doPlayback(itemid, dbid)
                                              File "/mnt/internal_sd/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.emby/resources/lib/entrypoint.py", line 46, in doPlayback
                                                pbutils.PlaybackUtils(item).play(itemId, dbId)
                                              File "/mnt/internal_sd/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.emby/resources/lib/playbackutils.py", line 57, in play
                                                playurl = playutils.getPlayUrl()
                                              File "/mnt/internal_sd/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.emby/resources/lib/playutils.py", line 113, in getPlayUrl
                                                playurl = self.transcoding()
                                              File "/mnt/internal_sd/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.emby/resources/lib/playutils.py", line 347, in transcoding
                                                if self.resolutionConflict():
                                              File "/mnt/internal_sd/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.emby/resources/lib/playutils.py", line 685, in resolutionConflict
                                                videoRes = self.getVideoResolution()
                                              File "/mnt/internal_sd/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.emby/resources/lib/playutils.py", line 701, in getVideoResolution
                                                return {'width' : self.item['MediaStreams'][0]['Width'],
                                            IndexError: list index out of range
                                            -->End of Python script error report<--
08:22:41.912 T:18446744072287576416   DEBUG: ------ Window Init (DialogNotification.xml) ------
08:22:42.163 T:18446744072445157928 WARNING: CPythonInvoker(328, /mnt/internal_sd/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.emby.tvshows/default.py): the python script "/mnt/internal_sd/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.emby.tvshows/default.py" has left several classes in memory that we couldn't clean up. The classes include: N9XBMCAddon9xbmcaddon5AddonE
08:22:42.163 T:18446744072445157928    INFO: Python script stopped
08:22:42.163 T:18446744072445157928   DEBUG: Thread LanguageInvoker 18446744072445157928 terminating
08:22:42.169 T:18446744072332583976   DEBUG: Thread scriptobs 18446744072332583976 terminating
```

First up, it looks like Emby is detecting it as 3064P, rather than 1080P. It then bombs out trying to get the stream dimensions.

**Comment by angelblue05** [comment]
Thanks, will look into it and post a new build shortly.

**Comment by angelblue05** [comment]
Should be resolved with 2.3.58.

**Comment by deece** [comment]
Confirmed fixed in 2.3.58, thanks :)
